src/Claim.ts

```diff
@@ -1,42 +1,62 @@
 /* tslint:disable:no-relative-imports */
 import * as bitcore from 'bitcore-lib'
 import * as crypto from 'crypto'
-
+import { canonize, expand } from 'jsonld'
 import { IllegalArgumentException } from './Exceptions'
```

Comment:
Please leave an empty line between 3rd party imports and imports of the same project

Reply:
ack